web/src/user_status.ts

```diff
@@ -5,6 +5,14 @@ import * as emoji from "./emoji";
 import type {EmojiRenderingDetails} from "./emoji";
 import {user_settings} from "./user_settings";
 
+type IsEmojiInfoPresent =
+    | {
+          type: "emoji_info_absent";
+      }
+    | ({
+          type: "emoji_info_present";
+      } & UserStatusEmojiInfo);
```

Comment:
What's the thinking around this name? It seems rather strange.

Reply:
Dropped the commit following the given [review comment](https://github.com/zulip/zulip/pull/28996#discussion_r1505211067).